doc/modules/mod_jingle_sip.md

```diff
@@ -42,3 +104,168 @@ MongooseIM 2.2.x packages are built with OTP 19.3, so they include Jingle/SIP su
+    |                                  |                     |   | sid: 456              | |
+    |                                  | 200 OK              |   |-----------------------| |
+    |                                  |-------------------->|                             |
+    |                                  | --------------\     |                             |
+    |                                  |-| from: UserA |     |                             |
+    |                                  | | to: UserB   |     |                             |
+    |                                  | | sid: 456    |     |                             |
+    |                                  | |-------------|     | find corresponding call     |
+    |                                  |                     |------------------------     |
+    |                                  |                     |                       |     |
+    |                                  |                     |<-----------------------     |
+    |                                  |                     |                             |
+    |                                  |              200 OK |                             |
+    |                                  |<--------------------|                             |
+    |                                  |     --------------\ |                             |
+    |                                  |     | from: UserA |-|                             |
+    |                                  |     | to: UserB   | |                             |
+    |                                  |     | sid: 123    | |                             |
+    |        session-accept from UserB |     |-------------| |                             |
+    |<---------------------------------|                     |                             |
+    |                                  |                     |                             |
+```
+
+<!---
+object UserA MongooseIM SIPProxy UserB
+UserB->MongooseIM: session-accpet to UserA
+note left of UserB: Jingle stanza \n action:session-accept\nsid: 456
+MongooseIM->SIPProxy: 200 OK
+note right of MongooseIM:from: UserA\nto: UserB\nsid: 456
+SIPProxy->SIPProxy: find corresponding call
+SIPProxy->MongooseIM: 200 OK
+note left of SIPProxy: from: UserA\nto: UserB\nsid: 123
+MongooseIM->UserA: session-accept from UserB
+-->
+
+##### 1.3 Terminating a call
+
+Any Jingle session (accepted or not) can be terminated by sending a Jingle stanza with action `session-terminate` and a reason.
+In the SIP world it's more complex.
+See the following examples for more information.
+
+###### 1.3.1 Terminating an accepted call
+
+The easiest scenario is when the call was accepted as in [1.2](#12-signaling-session-accept-to-other-xmpp-user-via-sip-proxy).
+In this case one of the users sends a `session-terminate` Jingle action with reason `success`.
```

Comment:
In this case one of the users sends a `session-terminate` Jingle action with a reason `success`.